content/tokio/topics/shutdown.md

```diff
@@ -129,48 +129,45 @@ a shutdown message on a connection.
 ## Waiting for things to finish shutting down
 
 Once you have told other tasks to shut down, you will need to wait for them to
-finish. The easiest way to do this is to use [an mpsc channel] where, instead of
-sending messages, you wait for the channel to be closed, which happens when
-every sender has been dropped.
+finish. One easy way to do so is to use a [task tracker]. A task tracker is a
+collection of tasks. The [wait] method of the task tracker gives you a future
+which resolves only after all of its contained futures have resolved **and** it
+has been closed.
```

Comment:
Instead of using "it", let's use "task tracker" to reduce the chance for confusion.
```suggestion
which resolves only after all of its contained futures have resolved **and** the
task tracker has been closed.
```